.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,123 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
```

Comment:
this file and the other one from idea shouldn't be committed & pushed. Please resolve your branch. :)
![image](https://user-images.githubusercontent.com/88314300/179957804-e2d162a0-524f-47c2-9acb-eee7322c00a2.png)